content/news/playtest-20150405.md

```diff
@@ -0,0 +1,16 @@
+We have received lots of feedback on our last playtest, for which we are very grateful. Let me take this opportunity to thank all our testers!
+
+In fact, we have received so much feedback that we felt it was worth it to do another, hopefully last, playtest before the release.
+
+This new playtest addresses most of the issues that have been reported. In particular, these are:
+
+  - The balancing in the Red Alert mod has received some more touch-ups.
```

Comment:
While this is certainly true, I still have a bit of work to do on this and some people might think that the balance has been finalised with this playtest :s